intrinsic_sdk_cmake/cmake/sdk.cmake

```diff
@@ -55,7 +55,8 @@ list(FILTER intrinsic_SRCS EXCLUDE REGEX "/intrinsic/icon/utils/metrics_logger.c
 list(FILTER intrinsic_SRCS EXCLUDE REGEX "/intrinsic/icon/interprocess/remote_trigger")
 list(FILTER intrinsic_SRCS EXCLUDE REGEX "/intrinsic/icon/interprocess/shared_memory_lockstep")
 list(FILTER intrinsic_SRCS EXCLUDE REGEX "/intrinsic/icon/interprocess/shared_memory_manager")
-list(FILTER intrinsic_SRCS EXCLUDE REGEX "/intrinsic/platform/pubsub/")
+# list(FILTER intrinsic_SRCS EXCLUDE REGEX "/intrinsic/platform/pubsub/")
```

Comment:
Remove?

Reply:
Yeah, I should do that, sorry vestigial.